pkg/ocm/client.go

```diff
@@ -117,13 +117,15 @@ func (c *Client) newConnection() error {
 
 // AuthPayload defines the structure of the User
 type AuthPayload struct {
-	Username     string `json:"username"`
-	FirstName    string `json:"first_name"`
-	LastName     string `json:"last_name"`
-	Organization string `json:"org_id"`
-	Email        string `json:"email"`
-	Issuer       string `json:"iss"`
-	ClientID     string `json:"clientId"`
-	IsAdmin      bool   `json:"is_admin"`
-	IsUser       bool   `json:"is_user"`
+	Username        string   `json:"username"`
+	FirstName       string   `json:"first_name"`
+	LastName        string   `json:"last_name"`
+	Organization    string   `json:"org_id"`
+	Email           string   `json:"email"`
+	Issuer          string   `json:"iss"`
+	ClientID        string   `json:"clientId"`
+	IsAdmin         bool     `json:"is_admin"`
+	IsReadOnlyAdmin bool     `json:"is_read_only_admin"`
+	IsUser          bool     `json:"is_user"`
```

Comment:
doesn't make sense to have three of them since user can be one or the other.
Maybe this should be en enum?